psi4/src/core.cc

```diff
@@ -569,6 +569,9 @@ void throw_deprecation_errors(std::string const& key, std::string const& module
                 ". All instances of 'dcft' should be replaced with 'dct'. The method was renamed in v1.4.",
             __FILE__, __LINE__);
     }
+    if (module == "SCF" && key == "DIIS_MIN_VECS") {
+        py_psi_print_out("WARNING!\n\tRemove keyword DIIS_MIN_VECS! This keyword does nothing. Using it will raise an error in v1.7.\n");
+    }
```

Comment:
I suspect that every use of diis_min_vecs (even post-scf ones that will remain legit) will hit this error during the scf part unless user specified module with `set dfocc diis_min_vecs`.

I don't have a great solution. Could tone down the warning to `WARNING!\n\tKeyword DIIS_MIN_VECS for the SCF module is deprecated (probably has not worked in a while). Directing it toward the SCF module will raise an error in v1.7.`. That's awkward enough that the read_options-only comment might be better.

Reply:
Incorrect.

My tests say the warning only fires if the user explicitly specifies this as an SCF option. It goes silent if it set as a global option or specific to a non-SCF module.

Reply:
oh, that's great!